Source/Core/Core/CMakeLists.txt

```
@@ -656,6 +656,10 @@ PRIVATE
   ZLIB::ZLIB
 )
 
+if(ENABLE_HWDB)
+  target_link_libraries(core PRIVATE udev)
+endif()
+
```

I'm not very familiar with CMake but based on this [other CMakeLists.txt file](https://github.com/dolphin-emu/dolphin/blob/165ce40f5d01c132e2c5b4b5671b89d237152dbc/Source/Core/InputCommon/CMakeLists.txt#L151-L167), shouldn't you check `LIBUDEV_FOUND` and target `${LIBUDEV_LIBRARY}` instead?

```suggestion
if(LIBUDEV_FOUND)
  target_link_libraries(core PRIVATE ${LIBUDEV_LIBRARY})
endif()

```

---

This seems to be the cleaner approach, yes, I'll do that